Part3_InterruptRoutingSetupInAChipset.html

```diff
@@ -3,24 +3,24 @@
 <html lang="en">
 <head>
 	<meta charset="utf-8">
-	<title>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset on the example of Coreboot</title>
+	<title>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset using coreboot</title>
 </head>
 
 <body>
-	<h1>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset on the example of Coreboot</h1>
+	<h1>External Interrupts in the x86 system. Part 3. Interrupt routing setup in a chipset using coreboot</h1>
 
 	<p>We continue to investigate external device interrupt routing setup in the x86 system.<p/>
 
-	<p>In the Part 1 (<a href="https://habr.com/ru/post/446312/">Interrupt controller evolution</a>) we've looked at the theory behind interrupt controllers and all the necessary terminology, in the Part 2 (<a href="???">Linux kernel boot options</a>) we've looked in practice how OS choses between different interrupt controllers. In this part we will investigate how does the BIOS set IRQ to the interrupt controllers routing in a chipset.<p/>
+	<p>In Part 1 (<a href="https://habr.com/ru/post/446312/">Interrupt controller evolution</a>) we looked at the theory behind interrupt controllers and all the necessary terminology. In the Part 2 (<a href="???">Linux kernel boot options</a>) we looked at how in practice the OS chooses between different interrupt controllers. In this part we will investigate how the BIOS sets IRQ to the interrupt controllers routing in a chipset.<p/>
```

Comment:
If 'the' is not needed when we talk about different parts of the article, it seems like you've messed 'the' in a sentence "In the Part 2 ..."

Reply:
Oops thanks.